library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt

```diff
@@ -240,19 +261,19 @@ class DonutProgressView @JvmOverloads constructor(
         return thisLine + previousLine
     }
 
-    private fun hasEntriesForCategory(category: String) =
-        entries.firstOrNull { it.category == category } != null
+    private fun hasEntriesForDataset(dataset: String) =
+        data.indexOfFirst { it.name == dataset } > -1
 
     private fun animateLine(
         line: DonutProgressLine,
         to: Float,
         animationEnded: (() -> Unit)? = null
     ): ValueAnimator {
-        return ValueAnimator.ofFloat(line.length, to).apply {
+        return ValueAnimator.ofFloat(line.mLength, to).apply {
             duration = ANIM_DURATION_MS
             interpolator = DecelerateInterpolator(INTERPOLATOR_FACTOR)
             addUpdateListener {
-                line.length = it.animatedValue as Float
+                line.mLength = it.animatedValue as Float
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/0780303dd6c89e1b717a798606795458927c0346/library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt#L276"></span>it.animatedValue cannot be safely cast to Float.</td>
    </tr>
  </tbody>
</table>